build.gradle

```diff
@@ -1,229 +1,388 @@
+                                artifactId "dependency-check-maven"
+                                version "1.2.11"
+                                configuration {
+                                    skipProvidedScope "true"
+                                    format "ALL"
+                                    // TODO: Turn this on once all deps are updated
+                                    //failBuildOnCVSS "7.0"
+                                }
+                            }
+                            plugin {
+                                groupId "org.apache.maven.plugins"
+                                artifactId "maven-compiler-plugin"
+                                version "3.3"
+                                configuration {
+                                    source "1.8"
+                                    target "1.8"
+                                    debug "true"
+                                    debuglevel "lines,vars,source"
+                                }
+                            }
+                            plugin {
+                                groupId "org.apache.maven.plugins"
+                                artifactId "maven-surefire-plugin"
+                                version "2.12.4"
+                            }
+                            plugin {
+                                groupId "org.apache.maven.plugins"
+                                artifactId "maven-assembly-plugin"
+                                version "2.4"
+                                configuration {
+                                    descriptors {
+                                        // To be evaluated by Maven
+                                        descriptor "\${project.basedir}/assembly.xml"
+                                    }
+                                }
+                                executions {
+                                    execution {
+                                        id "create-archive"
+                                        phase "package"
+                                        goals {
+                                            goal "single"
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        }.withXml {
+            asNode().getAt("properties")[0].appendNode("project.eclipse.directory", "\${project.basedir}/target-eclipse")
+            
+            /* NOTE: necessary b/c gradle doesn"t appear to expose activeByDefault. */
+            asNode().get("profiles")[0].get("profile")[0].appendNode("activation").replaceNode {
+                activation {
+                    activeByDefault "true"
+                }
+            }
+        }.writeTo("pom.xml")
+    }
+}
+
+wrapper {
+    gradleVersion = "5.3"
+    distributionType = Wrapper.DistributionType.ALL
+}
```

Comment:
Please add a newline at the end of the file here.

Reply:
Done.